tests/test_architecture.py

```diff
@@ -246,3 +365,26 @@ def test_one_hot_encoding_02(self):
         exp_value = X.values.shape[1]
 
         assert mfe._custom_args_ft["N"].shape[1] == exp_value
+
+
+class TestMemoryError:
```

Comment:
I am thinking if it would be not interesting to create a class TestWarning like this one

Reply:
Could you please elaborate a lit bit more? I am not sure about what this TestWarning class would be about.

Reply:
For example, adding the 'test_feature_warningx' to the TestWarning, and all of the Warnings architectural tests.

Reply:
I think I got it. I'll work on it also.